### ISSUE TITLE
Current DJ's avatar should only display once

### ISSUE THREAD

**Issue by eojthebrave** [ClockworkNet/cwfm#4]
**Problem**

Right now, if you're the DJ, on your display, your avatar shows up twice. If you're just an observer in the room however the DJ's avatar is correctly only listed once.

![image](https://cloud.githubusercontent.com/assets/17613/3633838/37cd4918-0eff-11e4-8075-f8bc3235c625.png)

**Steps to repeat**
- Login
- Enter a room
- Start DJing

**QA** Ensure that when you start DJing your avater is only listed once in the sidebar.

**Comment by eojthebrave** [comment]
This happens because the code in cwfm.room.js for the `addMe()` function doesn't check to see if a user has already been added to the dj list before adding them. Which ... they have, because they are getting added by the code that gets executed when the socket broadcasts 'dj.joined'.

I think we should just replace `addMe()` with a call to `addUser()`. Which, I did in the pull request #8.

In my testing this is working.
